library/alpine

```diff
@@ -1,9 +1,10 @@
 # maintainer: Glider Labs <team@gliderlabs.com> (@gliderlabs)
 
-3.1: git://github.com/gliderlabs/docker-alpine@8179fe722094b8724b7e9eb1d573173baf6ca977 versions/library-3.1
-3.2: git://github.com/gliderlabs/docker-alpine@5636e9a91b543fa507271fbbc9b5d1db59ce986f versions/library-3.2
-3.3: git://github.com/gliderlabs/docker-alpine@bc90ea0c90880b44cc7f405cbe6ec27ed453fda7 versions/library-3.3
-3.4: git://github.com/gliderlabs/docker-alpine@a83f6c797a617f580ff717efe9e53543f7beca43 versions/library-3.4
-latest: git://github.com/gliderlabs/docker-alpine@a83f6c797a617f580ff717efe9e53543f7beca43 versions/library-3.4
-edge: git://github.com/gliderlabs/docker-alpine@64804257e6997cdc88c39e949b6a9df492621691 versions/library-edge
+3.1: git://github.com/gliderlabs/docker-alpine@6a7732123b1e22c836e7ff538828ecf9eb0ebbb3 versions/library-3.1
+3.2: git://github.com/gliderlabs/docker-alpine@39157ab3587065a0e0928663dea293ae43110cd9 versions/library-3.2
+3.3: git://github.com/gliderlabs/docker-alpine@62168e0f780d64f2124a7be94ed31991a31ae5ad versions/library-3.3
+3.4: git://github.com/gliderlabs/docker-alpine@71c9ce882016759bcc0d47f372d20a1b6ec93cd5 versions/library-3.4
+latest: git://github.com/gliderlabs/docker-alpine@71c9ce882016759bcc0d47f372d20a1b6ec93cd5 versions/library-3.4
```

Comment:
yes, it should. Just wanted push the first image and double check that it works before making a change that affects alot of users.